riotfile.py

```diff
@@ -6,13 +6,22 @@
         Venv(
             name="test",
             command="pytest {cmdargs}",
-            pys=[3.6, 3.7, 3.8, 3.9],
             pkgs={
+                "click": latest,
                 "pytest": latest,
                 "pytest-cov": latest,
                 "mock": latest,
                 "typing-extensions": latest,
             },
+            venvs=[
+                Venv(
+                    pys=[3.6],
+                    pkgs={"dataclasses": latest},
```

Comment:
why didn't this fail before?

Reply:
Was wondering the same thing as the dependency on `click` was also missing. The only explanation that I could think of is that perhaps the site where riot was being installed leaked into the task runs. Now this doesn't seem to happen 🙂

Reply:
🤔  in fact these shouldn't be needed now that we also set the base venv in the `PYTHONPATH`, since those come with the installation of the dev package. Will investigate and fix

Reply:
@brettlangdon this is now fixed!